### PR TITLE
[igraph] update to 0.9.5

### PR DESCRIPTION
**Pull request by szhorvat**
**Describe the pull request**

- #### What does your PR fix?  
  Update to 0.9.5

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  all, no

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/igraph/portfile.cmake [hunk 1]
```diff
@@ -32,9 +32,8 @@ elseif (VCPKG_TARGET_IS_WINDOWS OR VCPKG_TARGET_IS_UWP)
     endif()
 endif()
 
-vcpkg_configure_cmake(
+vcpkg_cmake_configure(
     SOURCE_PATH ${SOURCE_PATH}
```
Comment (JonLiu1993) [inline ports/igraph/portfile.cmake:36]:
```suggestion
    SOURCE_PATH "${SOURCE_PATH}"
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by szhorvat** [comment]
@JonLiu1993 Thanks for the review! CI checks have passed, this PR is now complete.

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
All features are tested successfully in the following triplet:
- x86-windows
- x64-windows
- x64-windows-static

**Comment by szhorvat** [comment]
@JonLiu1993 Could you please answer my question from here? https://github.com/microsoft/vcpkg/pull/17683#issuecomment-834119358, https://github.com/microsoft/vcpkg/pull/18213#issuecomment-852020227

> When using `${FEATURE_OPTIONS}`, vcpkg adds `-DSOMEFLAG=ON` when an option is enabled. Does it also add `-DSOMEFLAG=OFF` when the option is not enabled?  Is there an easy way to see the CMake invocation, so I can determine this for myself?

**Comment by JonLiu1993** [comment]
@szhorvat ,If the options are set to OFF by default, we can set -DSOMEFLAG=OFF in ${FEATURE_OPTIONS} or not listed here, but if we need to enable this option, we need to modify -DSOMEFLAG=ON or add -DSOMEFLAG= ON, when to set it to ON, we need to analyze and judge according to the specific situation

**Pull request merged**